ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+
+Install And Verify TrustyAI Service
+    [Documentation]    Install TrustyAI service CRD and verify that TrustyAI resources have spun up
+    Deploy Custom Resource    kind=TrustyAIService    namespace=${PRJ_TITLE}
+    ...    filepath=${TRUSTYAI_CR_FILEPATH}
+    Sleep   15s
+    ${return_code}    ${output}    Run And Return Rc And Output   oc get route trustyai-service -n ${PRJ_TITLE} --template={{.spec.host}}
+    Should Be Equal As Integers    ${return_code}	 0
+    Set Suite Variable    ${TRUSTY_ROUTE}    https://${output}
+
+Delete TrustyAI Service
+    [Documentation]    Delete TrustyAI service CRD
+    OpenshiftLibrary.Oc Delete    kind=TrustyAIService    namespace=${PRJ_TITLE}    name=trustyai-service
+
+Verify Model Is Registered with TrustyAI Service
+    [Documentation]    Verify the deployed model is registered with TrustyAI Service by verifying MM_PAYLOAD_PROCESSORS
+    ...                value is the TrustyAI service route
+    ${MM_PAYLOAD_PROCESSORS_Expected}=    Set Variable    http://trustyai-service.${PRJ_TITLE}.svc.cluster.local/consumer/kserve/v2
+    Wait Until Keyword Succeeds  3 min  20 sec  Verify One Model Serving Pod Exists   ${PRJ_TITLE}    label_selector=modelmesh-service=modelmesh-serving
+    ${return_code}    ${podname}    Run And Return Rc And Output  oc get pods -n ${PRJ_TITLE} -o json | jq '.items[].metadata.name' | grep 'modelmesh-serving'
+    Log    Serving Runtime Podname: ${podname}
+    ${MM_PAYLOAD_PROCESSORS_Actual} =  Run  oc get pod ${podname} -n ${PRJ_TITLE} -o json | jq '.spec.containers[0].env[] | select(.name=="MM_PAYLOAD_PROCESSORS") | .value'
+    ${MM_PAYLOAD_PROCESSORS_Actual} =    Strip String    ${MM_PAYLOAD_PROCESSORS_Actual}    characters="
+    Should Be Equal  ${MM_PAYLOAD_PROCESSORS_Actual.strip()}  ${MM_PAYLOAD_PROCESSORS_Expected.strip()}
+
+Verify One Model Serving Pod Exists
+    [Documentation]    Verifies that serving runtime pods have been stabilized and only 1 pod exists
+    [Arguments]    ${PRJ_TITLE}     ${label_selector}
+    ${return_code}    ${pod_number}    Run And Return Rc And Output   oc get pod -n ${PRJ_TITLE} -l ${label_selector} | tail -n +2 | wc -l
+    ${pod_numbers}    Split String     ${pod_number}
+    Should Be Equal    ${pod_numbers}[0]   1
+
+#Verify MM_PAYLOAD_PROCESSORS Environment Variable Is Removed
+#    [Documentation]    Verify that the MM_PAYLOAD_PROCESSORS is deleted when TrustyAI service is removed
+#    ${podname} =  Run  oc get pods -n ${PRJ_TITLE} -o json | jq '.items[].metadata.name' | grep 'modelmesh-serving'
+#
+
+Send Batch Inference Data to Model
+    [Documentation]    Send Batch Inference data to the already deployed model using Curl commands
+    [Arguments]        ${model_name}   ${project_name}
+    ${url}=    Get Model Route via CLI    ${model_name}   ${project_name}
+    #To be removed PWD Command
+    ${rc}    ${pwd}=    Run And Return Rc And Output    echo $PWD
+    FOR    ${counter}    IN RANGE    1    5
```

Comment:
the PR adds up to 13 json files for batches, do we need only 5?

Reply:
I will be using the remaining for creating tests via UI and other non-admin user tests.